### PR TITLE
update, delete, show api endpoint

### PR DESCRIPTION
**Pull request by perryr16**
# What's this PR do?
* update, create, delete endpoints
# Where should the reviewer start?
* CampaignView
# How should this be manually tested?
* no tests yet
* `/api/v1/campaigns/<campaign_id>` verb: `put`, `delete`, `get`
# Any background context you want to provide?
* more endpoints 
# What are the relevant tickets?

### CODE REVIEW

File: eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py [hunk 1]
```diff
@@ -29,34 +29,35 @@ def custom_response(res, status_code):
...
+  data = campaign_schema.load(req_data, partial=True)
+  # error not functional yet
+  error = None
   if error:
     return custom_response(error, 400)
   
   campaign = Campaign.get_one_campaign(campaign_id)
   campaign.update(data)
-  campaign_data = campaign_schema.dump(campaign).data
+  campaign_data = campaign_schema.dump(campaign)
```
Comment (perryr16) [inline eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py:52]: cant use `(campaign).data`

File: eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py [hunk 2]
```diff
@@ -29,34 +29,35 @@ def custom_response(res, status_code):
...
   return custom_response(campaign_data, 200)
 
-# NOT YET FUNCTIONAL
-@campaign_api.route('/<int:campaign_id>', methods=["DELTE"])
+
+@campaign_api.route('/<int:campaign_id>', methods=["DELETE"])
```
Comment (perryr16) [inline eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py:56]: misspelled `DELETE`

**Review by AntonioJacksonII — APPROVED** [review]
Great job getting the Campaign CRUD routes set up and functioning!

File: eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py [hunk 3]
```diff
@@ -29,34 +29,35 @@ def custom_response(res, status_code):
...
 def delete(campaign_id):
   campaign = Campaign.get_one_campaign(campaign_id)
-  campaign_data = campaign_data.dump(campaign).data
+  campaign.delete()
+  campaign_data = campaign_schema.dump(campaign)
   return custom_response(campaign_data, 200)
```
Comment (takeller) [inline eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py:61]: Delete isn't working when campaign has dependent requests. We'll have to look into dependent destroy

**Review by takeller — APPROVED** [review]

**Pull request merged**